### DISCUSSION TITLE
Notification not show

### DISCUSSION THREAD

**Discussion by xiaomlove** [filamentphp/filament#4769] [Help] [answered]
### Filament: 2.16.36

After I upgraded from 2.14.2 to 2.16.36, I have executed 
```
php artisan livewire:publish --assets
php artisan filament:upgrade
```
I still encountered the following problems.

### Issue 1: There are two warning errors: index.js:85 Livewire: Multiple root elements detected.
The first warning: 
![QQ截图20221101025706](https://user-images.githubusercontent.com/8530353/199087842-e1fae86d-c592-4b45-b2fc-727d7d692824.png)

The second warning:
![QQ截图20221101025826](https://user-images.githubusercontent.com/8530353/199088118-e5ff6dde-5fda-4473-9f0b-20b3107bf1a7.png)

### Issue 2: Notification alerts are completely missing. 
The results returned by `/livewire/message/filament.core.notifications` are empty for effects.dirty/html. I found this video：https://www.youtube.com/watch?v=NY-HdGHSDKE&ab_channel=Laraveller  through a Google search, and after following the author's instructions and running npm run dev, it didn't work. In fact, I have a question that this npm run dev will encode and generate js/app.js + css/app.css, but the actual filament introduces filament/assets/app.css + filament/assets/app.js, is there any relationship between the two?

Is issue 2 caused by issue 1? Thanks for everyone's help!

**Comment by Z3d0X** [comment]
Delete `/resources/views/vendor`, `php artisan filament:upgrade` & rebuild your assets (`npm run production` or `npm run build`). 

https://filamentphp.com/blog/publishing-views-in-laravel

**Comment by xiaomlove** [comment]
@Z3d0X I haven't publish views, and already upgrade, also I had `npm run production` , it will build  ` js/app.js` + `css/app.css`, but filament doesn't introduces them...filament introduces `filament/assets/app.css` + `filament/assets/app.js`

**Comment by xiaomlove** [comment, marked as answer]
Finally I know what the problem is! notifications is using session to store messages, and I modified the `middleware_group` in the livewire configuration, which does not contain the `StartSession` middleware! Thanks for all the help.
